libs/net/dal/Configuration/SourceConfiguration.cs

```diff
@@ -15,6 +15,7 @@ public override void Configure(EntityTypeBuilder<Source> builder)
         builder.Property(m => m.ShortName).IsRequired().HasMaxLength(100).HasDefaultValueSql("''");
         builder.Property(m => m.Description).IsRequired().HasMaxLength(2000).HasDefaultValueSql("''");
         builder.Property(m => m.OwnerId);
+        builder.Property(m => m.ProductId);
```

Comment:
You should define the relationship as this is a foreign key.  See the examples below.